roles/logs_toprm/tasks/logstoprm.yml

```diff
@@ -0,0 +1,85 @@
+---
+- name: Create a list of all names
+  set_fact:
+    all_log_servers: "{{ logs_server_public_networks | dict2items | map(attribute='value') | flatten }}"
+
+- name: Print every logs server name and public IP
+  ansible.builtin.debug:
+    msg: "{{ item.name }} - {{ '%-15s' | format(all_ip_addresses[item['name']][item['network']].address) }}"
+  loop: "{{ all_log_servers }}"
+
+- name: 'Get {{ logs_toprm_chaperone_user }} home directory'
+  ansible.builtin.shell: "set -o pipefail && getent passwd {{ logs_toprm_chaperone_user }}  | awk -F: '{ print $6 }'"
+  changed_when: false
+  register: home_path
+
+- name: 'Install logstoprm cron .sh script.'
+  ansible.builtin.template:
+    src: logstoprm_cron.sh.j2
+    dest: "{{ home_path.stdout }}/logstoprm_cron.sh"
+    owner: "{{ logs_toprm_chaperone_user }}"
+    group: "{{ logs_toprm_chaperone_user }}"
+    mode: '0750'
+  become: true
+
+- name: "Check that the {{ logs_toprm_chaperone_user }} user has already created keys"
+  ansible.builtin.stat:
+    path: "{{ home_path.stdout }}/.ssh/id_ed25519"
+  register: key_details
+  become: true
+  become_user: "{{ logs_toprm_chaperone_user }}"
```

Comment:
```suggestion
```

Reply:
This task appear to be useless as the result in `key_details` is not used anywhere. When the following `community.crypto.openssh_keypair` task is idempotent there is also no need to check if a key pair already exists. So unless there is a bug in `community.crypto.openssh_keypair` we do not need the `ansible.builtin.stat` task here, right?